mobile/screens/PeriodCalendarScreen.js

```diff
@@ -145,6 +150,171 @@ const PeriodCalendarScreen = ({ props }) => {
         .catch(error => {console.log(error)})
     }
 
+    const getDateFromDiff = (diff, date=currDateObject) => {
+        let dayAtDiff = new Date(date.getTime());
+        dayAtDiff.setDate(dayAtDiff.getDate() + diff);
+        return dayAtDiff;
+    }
+
+    // TODO: make async and actually fetch
+    function fetchNotesForDate(date) {
+        return "Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit";
+    }
+
+    // fetch methods here will return an object for each day in the week
+    const fetchWeekFlow = (startDate) => {
```

Comment:
Ideally, I would only have one fetch function. Backend should return as a JSON object:
`<date>: {<symptom_0>: <values>, <symptom_1>: <values>, ...}`
but can update later